src/docs/sdk/features.mdx

```diff
@@ -169,6 +169,6 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 
 - operation: `http.client`
 - description: `$METHOD $url` (uppercase HTTP method), e.g. `GET https://sentry.io`
-- HTTP request must be enhanced with a [`sentry-trace` HTTP header](/sdk/performance/#header-sentry-trace)
+- HTTP requests must be enhanced with a [`sentry-trace` HTTP header](/sdk/performance/#header-sentry-trace) to support [distributed tracing](https://docs.sentry.io/platforms/java/performance/instrumentation/custom-instrumentation/#distributed-tracing
```

Comment:
this is the common docs but linking to the java platform docs, not sure if its the right away, we probably have a common section that explains the `sentry-trace` header, its part of the spec